tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/SDKBuildPropertiesResourceListener.java

```diff
@@ -119,34 +156,25 @@ public void resourceChanged( final IResourceChangeEvent event )
         try
         {
             final IResourceDelta delta = event.getDelta();
-            final SDK sdk = SDKUtil.getWorkspaceSDK();
 
-            if( delta != null && sdk != null )
+            if( delta != null )
             {
                 for( IResourceDelta child : delta.getAffectedChildren() )
                 {
-                    final IPath deltaLocation = child.getResource().getLocation();
+                    final IResourceDelta[] sdkChangedFiles =
```

Comment:
whiile the sdk project has been deleted, the sdkBuildPropertiesListener monitor the  post event, so we can't get sdk instance in that moment. I will try to change it.